backend-es/test/snapshots-out/Snapshot.EffectBind01.js

```diff
@@ -1,11 +1,9 @@
 import * as Effect$dConsole from "../Effect.Console/index.js";
-const test1 = /* #__PURE__ */ (() => {
-  const $0 = Effect$dConsole.log("1");
-  return () => {
-    $0();
-    const value = Effect$dConsole.log("2")();
-    Effect$dConsole.log("3")();
-    return value;
-  };
-})();
-export {test1};
+const test1$0 = /* #__PURE__ */ Effect$dConsole.log("1");
+const test1 = () => {
+  test1$0();
+  Effect$dConsole.log("2")();
+  Effect$dConsole.log("3")();
+  return value$1;
```

Comment:
This example seems to have incorrect codegen. It's returning a `value$1` which is never defined.

Reply:
Oh wow that's a killer regression, I missed it, thanks for spotting it. I'll fix it & then respond to the other comment after the 🐛 is squashed.

Reply:
Fixed 🚀 
I was missing a case for `EffectBind`.